### ISSUE TITLE
Synchronize columns with database find column data types, but doesn't save to model.yml file.

### ISSUE THREAD

**Issue by JustGitting** [AltimateAI/vscode-dbt-power-user#809]
### Expected behavior

In the "Documentation Editor" tab and selecting the "Synchronize columns with database" updates the column names and data types from the database.

For example:

my_name (varchar)
my_number (int)

Then pressing "Save Documentation" should update the model column names and DBT "[data_type](https://docs.getdbt.com/reference/resource-properties/columns)" field in the model.yml file.

The model.yml file should contain something like:

```
version: 2
models:
  - name: stg_test__my_model
    description: test table
    columns:
      - name: my_name
        description: "Contains a string"
        data_type: varchar
      - name: my_number
        description: "Contains a number"
        data_type: int
```
     

### Actual behavior

The data_type data is not saved in the model.yml file.

After pressing "Save Documentation", the model.yml is updated with column names, but not the data type information.

There are no errors or warnings given.

### Steps To Reproduce

1. Open Codium.
2. Select my_data_model.sql file.
3. Go to "Documentation Editor" tab at bottom.
4. Select "Synchronize columns with database"
5. Wait for table details to be updated from the database.
6. Select "Save Documentation"

### Log output/Screenshots

There were no errors or warnings.

### Operating System

Devuan GNU/Linux 4 (chimaera)

### dbt version

1.4.6

### dbt Adapter

dbt-sqlserver v1.4.3

### dbt Power User version

0.28.4

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by barskern** [comment]
Seeing the same issue in my project.

**Versions**:
dbt: 1.6.1
dbt Power User: 0.29.1
dbt-postgres: 1.6.1
OS: Windows 10

**Comment by barskern** [comment]
Some related lines of code:

https://github.com/AltimateAI/vscode-dbt-power-user/blob/45bcef5688829c378a16f87cbbebc558477a5901/src/webview_provider/docsEditPanel.ts#L542C57-L542C57

https://github.com/AltimateAI/vscode-dbt-power-user/blob/45bcef5688829c378a16f87cbbebc558477a5901/src/webview_provider/docsEditPanel.ts#L558-L567

**Comment by anandgupta42** [comment]
Thanks @barskern for the contribution. This has been released in v0.30.0
